composer.json

```diff
@@ -23,7 +23,7 @@
     "ext-json": "*",
     "contributte/psr7-http-message": "~0.7.0",
     "doctrine/annotations": "^1.13",
-    "koriym/attributes": "^1.0",
+    "koriym/attributes": "1.x-dev#2b0349d",
```

Comment:
Do you know when will be release stable version? 1.1 or 1.0.1?

Reply:
Working on it https://github.com/koriym/Koriym.Attributes/pull/11#issuecomment-878144758 :)